lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -69,7 +79,7 @@ public function request( $resourceName, $getParams = [], $postData = [], $extraR
 		// Request options pre-processing.
 		$options = [
 			'method'          => 'GET',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "method" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)